config/deploy.js

```diff
@@ -40,7 +40,7 @@ module.exports = function(deployTarget) {
 
   if (deployTarget === 'production') {
 
-    const bucket = 'ember-ghost-w3css-blog-template';
+    const bucket = 'empress-blog-w3css-blog-template';
```

Comment:
If you're not using ember-cli-deploy (because you're using netlify) you should delete this and remove the ember-cli-deploy dependencies from this repo 👍 I am assuming that they just made it over because they were copy-pasted 🍝

Reply:
Yep. Copy & pasted.